### PR TITLE
Ipad avatar selection fix

### LINKED ISSUE #619: avatar selection does not work on ipad
avatar selection does not work on ipad, neither by the camera nor by the gallery. also the screen for cropping does not work, the preview is much too small.

tested on an ipad 2019 10.2

### PR DESCRIPTION
**Pull request by nayooti**
fixes #619 

Well, I figured out, that the bug is inside the `ALCamerViewController`-Library which hasn't got any updates for 6 months (pending PRs remain unanswered). The Ipad-bug was very well known there.

Easiest fix was to switch to a fork who addressed that issue and also claims to actively support the library, so we're using this fork ([`KKALCameraViewControlle`](https://github.com/kkieffer/ALCameraViewController)) now. 

I had to update all Pods which implied some minor changes in code.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ target 'deltachat-ios' do
 
   pod 'SwiftLint'
   pod 'SwiftFormat/CLI'
-  pod 'ALCameraViewController'
+	pod 'KK-ALCameraViewController'
```
Comment (cyBerta) [inline Podfile:11]: not a big deal, but the indentation is strange here

File: Pods/DBDebugToolkit/DBDebugToolkit/Resources/DBMenuTableViewController.storyboard [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15702" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="YH8-ay-XNT">
-    <device id="retina4_7" orientation="portrait" appearance="light"/>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="12118" systemVersion="16E195" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="YH8-ay-XNT">
```
Comment (cyBerta) [inline Pods/DBDebugToolkit/DBDebugToolkit/Resources/DBMenuTableViewController.storyboard:2]: What has the DBDebugToolkit todo with it?
I think you can reset these changes related to DBDebugToolkit?!

Comment (nayooti) [reply Pods/DBDebugToolkit/DBDebugToolkit/Resources/DBMenuTableViewController.storyboard:2]: I think there is no easy way for selective updates on specific pods other. Only solution I know out of the box would to add a specific version number on each pod we're using. But this way we won't be able to update all pods at once which I don't mind at all.

**Comment by cyBerta** [comment]
closing this in favor of #657

**Pull request closed without merging**